src/main/java/com/ge/predix/uaa/token/lib/FastTokenServicesCreator.java

```diff
@@ -16,9 +16,18 @@
 
 package com.ge.predix.uaa.token.lib;
 
+@Deprecated
 public class FastTokenServicesCreator {
 
     public FastTokenServices newInstance() {
-        return new FastTokenServices();
+        FastTokenServices fastTokenServices = new FastTokenServices();
```

Comment:
Instead of having to catch Exception here, consider changing FastTokenService to use a regular map in the initializer for tokenKeys (line 87).  It can then override to a expiring map, if afterPropertiesSet is called.  This file will not require any change other than marking as deprecated.

Reply:
I like that less change and more control. Practicality trumps idealism.